tests/atlas.phpt

```diff
@@ -3,6 +3,7 @@ Atlas Connectivity Tests
 --SKIPIF--
 <?php
 if (!file_exists('.travis.scripts/atlas-uris.txt')) { echo "skip Atlas URIs not found\n"; }
+if (filesize('.travis.scripts/atlas-uris.txt') < 10) { echo "skip Atlas URI file empty\n"; }
```

Comment:
This likely also belongs in a separate commit. I assume there was some error where decrypting failed and left a tiny file? If so, is there a better way to catch that in the Travis shell scripts instead of this curious conditional?

Reply:
Doing that in the Travis shell scripts was messy, and complicated, (and not working). As we're only doing this once, I felt that a simple solution in the PHP skipif was more warranted.